controllers/administration.js

```diff
@@ -946,6 +946,24 @@ const getTeacherUpdateHandler = () => async function teacherUpdateHandler(req, r
 		});
 };
 
+router.post(
+	'/registrationlinkMail/',
+	permissionsHelper.permissionsChecker(['ADMIN_VIEW', 'TEACHER_CREATE'], 'or'),
+	generateRegistrationLink({}),
+	(req, res) => {
+		const email = req.body.email || req.body.toHash || '';
+		api(req).get('/users', { qs: { email }, $limit: 1 })
+			.then((users) => {
+				if (users.total === 1) {
+					sendMailHandler(users.data[0], req, res, true);
```

Comment:
users.data[0] should be tamplated into text and not send in plain json

Reply:
Don't know whether I understand the issue, but the `sendMailHandler` function takes a json user to send a mail template with this user's data.